docs/05.PORT-API.md

```diff
@@ -150,7 +150,17 @@ jerry_port_log (jerry_log_level_t level, /**< log level */
 {
   va_list args;
   va_start (args, format);
+#ifdef JERRY_DEBUGGER
```

Comment:
I am not sure we should add this basic example. What others think?

Reply:
I agree with @zherczeg. I'd do not add this to this.

Reply:
So just keep the old format in the api?

Reply:
the old example. This should be a educational example, not a full implementation.

Reply:
Fine, I'll just put it back to its old form then.